src/ray/rpc/client_call.h

```diff
@@ -123,16 +123,24 @@ class ClientCallManager {
   ///
   /// \param[in] main_service The main event loop, to which the callback functions will be
   /// posted.
-  explicit ClientCallManager(boost::asio::io_service &main_service)
-      : main_service_(main_service) {
+  explicit ClientCallManager(boost::asio::io_service &main_service, int num_threads = 8)
```

Comment:
Before we merge this, we should set the default back to 1 and then only set it to a higher number for the object manager (as this code is shared between a number of components).